allennlp/data/tokenizers/word_filter.py

```diff
@@ -73,5 +72,5 @@ def __init__(self):
                               "'", '"', '&', '$', '#', '@', '(', ')', '?'])
 
     @overrides
-    def should_keep_words(self, words: List[str]) -> List[bool]:
-        return [word not in self.stopwords for word in words]
+    def filter_words(self, words: List[Token]) -> List[Token]:
+        return [word for word in words if word.text not in self.stopwords]
```

Comment:
is there a casing issue here? self.stopwords is all lowercase, but work.text may not be?

Reply:
Thanks, good catch.

Reply:
Done.